src/app/friend-link/friend-link.component.css

```
@@ -0,0 +1,17 @@
+#graphic {
+    /* Fountain Pen SVG */
+    width: 30em;
+}
+
+#buttons > button {
+    /* For each button */
+    margin: 0 1em;
+    margin-top: 2em;
+    font-size: 1.1em;
+}
+
+#title-text {
+    /* We see you’re trying to sign your friend’s virtrolio... */
+    text-align: center;
+    font-size: 2em;
+}
```

No newline at end of file
```suggestion
}

```